$MirOS$

Fix their idea of COFF to be in sync
with <sys/exec_ecoff.h> (our kernel)

--- utils/8l/asm.c.orig	Fri Mar 31 01:38:42 2006
+++ utils/8l/asm.c	Fri Mar 31 03:01:22 2006
@@ -202,7 +202,7 @@ asmb(void)
 		lputl(0);			/* unix time stamp */
 		lputl(0);			/* symbol table */
 		lputl(0);			/* nsyms */
-		lputl(0x0003001c);		/* flags, sizeof a.out header */
+		lputl(0x00030020);		/* flags, sizeof a.out header */
 		/*
 		 * a.out header
 		 */
@@ -213,6 +213,7 @@ asmb(void)
 		lput(entryvalue());		/* va of entry */
 		lputl(INITTEXT);		/* text start */
 		lputl(INITDAT);			/* data start */
+		lputl(INITDAT+datsize);		/* bss start */
 		/*
 		 * text section header
 		 */
